src/pages/EditMonitor.vue

```diff
@@ -143,6 +146,53 @@
                                 </select>
                             </div>
 
+                            <!-- Kafka Brokers List -->
+                            <!-- Kafka Producer only -->
```

Comment:
Can we merge these blocks into one check? (better readable, see line 211 for what I mean)